bin/console-conf-wrapper

```diff
@@ -22,7 +22,14 @@ if [ -e /run/snapd-recovery-chooser-triggered ]; then
     fi
 fi
 
-if grep -q 'snapd_recovery_mode=install' /proc/cmdline ; then
+# always prefer to use modeevn if it is available
+if [ -e "/var/lib/snapd/modeenv" ]; then
```

Comment:
I don't know much about snapd recovery modes or this modeenv file, can you point me to something relevant?

My big question is on the notion that the kernel command line would not take precedence over the on-disk value, as the /proc/cmdline is presumed to be easier to change on the fly.  What's going on there?

Reply:
@valentindavid can you please comment about use of the kernel cmd line vs modeenv file?

Reply:
`snapd_recovery_mode=install` is to instruct `snap-booststrap` what mode to run. `/var/lib/snapd/modeenv` is the mode for snapd to run, which is set during install by `snap-boostrap` and maintained later by `snapd`. And this file contains the effective mode. So it is more correct than the kernel command line.

Unfortunately there is not `snap` command to ask the current mode.